src/settings/plugins/ifcfg-rh/nms-ifcfg-rh-reader.c

```diff
@@ -2275,12 +2274,15 @@ make_sriov_setting (shvarFile *ifcfg)
 		}
 	}
 
-	s_sriov = (NMSettingSriov *) nm_setting_sriov_new ();
-	g_object_set (s_sriov,
-	              NM_SETTING_SRIOV_TOTAL_VFS, total_vfs,
-	              NM_SETTING_SRIOV_VFS, vfs,
-	              NM_SETTING_SRIOV_AUTOPROBE_DRIVERS, (int) autoprobe_drivers,
-	              NULL);
+	if (enable || total_vfs || vfs) {
```

Comment:
same comment as for the writer below.

Reply:
For the reader I think we need a bit of logic to decide whether to instantiate a sriov setting or not.